tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/LabelSetMethod.scala

```diff
@@ -4,20 +4,20 @@ import com.linagora.tmail.james.jmap.json.LabelSerializer
 import com.linagora.tmail.james.jmap.method.CapabilityIdentifier.LINAGORA_LABEL
 import com.linagora.tmail.james.jmap.model.{LabelSetRequest, LabelSetResponse}
 import eu.timepit.refined.auto._
-import javax.inject.Inject
 import org.apache.james.jmap.core.CapabilityIdentifier.{CapabilityIdentifier, JMAP_CORE}
 import org.apache.james.jmap.core.Invocation.{Arguments, MethodName}
 import org.apache.james.jmap.core.{ClientId, Id, Invocation, ServerId, SessionTranslator, UuidState}
-import org.apache.james.jmap.json.ResponseSerializer
 import org.apache.james.jmap.method.{InvocationWithContext, MethodRequiringAccountId}
 import org.apache.james.jmap.routes.SessionSupplier
 import org.apache.james.mailbox.MailboxSession
 import org.apache.james.metrics.api.MetricFactory
 import org.reactivestreams.Publisher
-import play.api.libs.json.{JsError, JsObject, JsSuccess}
+import play.api.libs.json.JsObject
 import reactor.core.scala.publisher.SMono
 
-class LabelSetMethod @Inject() (createPerformer: LabelSetCreatePerformer,
+import javax.inject.Inject
```

Comment:
Please adapt the suggested Scala import layout.

Reply:
Let me update it
// I re-imported IDE setting a few days ago, I remember it work yesterday, I don't know what happened today